src/views/secondary-bar/tests/empty-modal.test.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * SPDX-FileCopyrightText: 2021 Zextras <https://www.zextras.com>
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import { act } from '@testing-library/react';
+import { rest } from 'msw';
+import { generateStore } from '../../../tests/generators/store';
+import { getSetupServerApi } from '../../../carbonio-ui-commons/test/jest-setup';
+import { folderAction } from '../../../store/actions/folder-action';
+import { ContactsFolder } from '../../../types/contact';
+import { FolderActionRequest } from '../../../types/soap';
+/**
+ * Test the address book Empty modal
+ */
+describe('Empty model', () => {
+	test('folder action', async () => {
```

Comment:
There is a typo in the _describe_ description.
Also the test description should be more detailed because, if the test will fail, it will be difficult to understand what went wrong